vaadin-spring/src/main/java/com/vaadin/flow/spring/SpringServlet.java

```diff
@@ -161,6 +161,11 @@ private Properties config(Properties initParameters) {
         properties.putAll(initParameters);
         PROPERTY_NAMES.forEach(property -> setProperty(property, properties));
 
+        // this property is reversed
+        if ("true".equalsIgnoreCase(
+                initParameters.getProperty("disable-xsrf-protection")))
+            properties.put("xsrf-protection-enabled", false);
```

Comment:
What does this code do ?

`InitParameters.SERVLET_PARAMETER_DISABLE_XSRF_PROTECTION` is an init parameter which is used in the deployment configuration.

`initParameters` are servlet init parameters.
Produced `properties` are used for deployment configuration . 
Deployment configuration doesn't know anything about `xsrf-protection-enabled`. 
So I don't understand why this code exists.

Reply:
I do not know, either. I remove it.